### PR TITLE
Documentation patrick

### PR DESCRIPTION
**Pull request by killje**
My documentation for all the items

### CODE REVIEW

File: Switchboard/public/app/scripts/factory/Events.js [hunk 1]
```diff
@@ -6,17 +6,30 @@
...
+
+
     /**
-    * A module representing Events.
-    * @exports factory/Events
-    */
+     * @description
+     * An angularJS factory.
+     * 
+     * This factory is a model that connects with the back end to retive data
```
Comment (bbusagroningen) [inline Switchboard/public/app/scripts/factory/Events.js]: typo

File: Switchboard/public/app/scripts/factory/Page.js [hunk 2]
```diff
@@ -37,22 +61,53 @@ define(['Registry', 'factory/Alert'], function (Registry) {
 			return parsedTitle;
 		};
 		
+        /**
+         * Sets wheter or not the page is still loading.
+         * 
+         * When its loading, a loading icon will apear and any content is hidden
```
Comment (bbusagroningen) [inline Switchboard/public/app/scripts/factory/Page.js]: typo

File: Switchboard/public/app/scripts/factory/Page.js [hunk 3]
```diff
@@ -37,22 +61,53 @@ define(['Registry', 'factory/Alert'], function (Registry) {
...
+         * untill loading is set to false
```
Comment (bbusagroningen) [inline Switchboard/public/app/scripts/factory/Page.js]: typo

File: Switchboard/public/app/scripts/factory/Page.js [hunk 4]
```diff
@@ -37,22 +61,53 @@ define(['Registry', 'factory/Alert'], function (Registry) {
...
+         * @param {boolean} isLoading
+         */
 		Page.setLoading = function(isLoading) {
 			loading = isLoading;
 		};
 		
+        /**
+         * Gets the loading state of the page
+         * 
+         * @returns {boolean} Wheter or not the page is loading or not
```
Comment (bbusagroningen) [inline Switchboard/public/app/scripts/factory/Page.js]: typo, and also phrase it like this:
Whether or not the page is loading ( you can ignore the "or not" part, that can be derived from the word whether already)

File: Switchboard/public/app/scripts/factory/Page.js [hunk 5]
```diff
@@ -37,22 +61,53 @@ define(['Registry', 'factory/Alert'], function (Registry) {
...
 			return loading;
 		};
 		
+        /**
+         * Binds data that can be used in the title as specified in 
+         * factory/Pages
+         * 
+         * @param {string} bindName The name for the data to bind
+         * @param {Object.<string, string>} bindValue The data which may
+         * contain title information
```
Comment (bbusagroningen) [inline Switchboard/public/app/scripts/factory/Page.js]: contains

File: Switchboard/public/app/scripts/factory/Resources.js [hunk 6]
```diff
@@ -1,12 +1,18 @@
...
-    * A module representing a resouseces.
-    * @exports factory/Resources
-    */
+
+    /**
+     * @class factory/Resources
+     * @description
+     * All information about what urls to use are here
+     * 
+     * @param {Object} $resource The angular resource service. This is a api to
```
Comment (bbusagroningen) [inline Switchboard/public/app/scripts/factory/Resources.js]: an

File: Switchboard/public/app/scripts/main.js [hunk 7]
```diff
@@ -1,11 +1,21 @@
+/*
+ * !@NOTE! This class is not loaded inside the test suit. When aading or
```
Comment (bbusagroningen) [inline Switchboard/public/app/scripts/main.js]: typo

File: Switchboard/public/app/scripts/Loader.js [hunk 8]
```diff
@@ -1,54 +1,65 @@
...
         /**
-         * Creates a link element from document
+         * @private
+         * @description
+         * Creates a link element from the path specified
          * 
-         * @param {String} path The path that the link has to revere to
-         * @returns {Element} The element created for the current document
+         * @param {string} path The path that the link has to revere to
+         * @returns {Element} The DOM element created for the current document
```
Comment (killje) [inline Switchboard/public/app/scripts/Loader.js]: Document Object Model
https://www.w3schools.com/js/js_htmldom.asp

Comment (bbusagroningen) [inline Switchboard/public/app/scripts/Loader.js]: Alright, my bad

File: Switchboard/public/app/scripts/factory/Page.js (hunk 5 above)
Comment (killje) [inline Switchboard/public/app/scripts/factory/Page.js]: i would say contain as its just one object "the data" and it does not sound right

Comment (bbusagroningen) [inline Switchboard/public/app/scripts/factory/Page.js]: you are right, data is actually singular

File: Switchboard/public/app/scripts/main.js [hunk 9]
```diff
@@ -1,5 +1,5 @@
 /*
- * !@NOTE! This class is not loaded inside the test suit. When aading or
+ * !@NOTE! This class is not loaded inside the test suit. When ading or
```
Comment (bbusagroningen) [inline Switchboard/public/app/scripts/main.js]: still typo

**Pull request merged**